include/uv/unix.h

```diff
@@ -326,6 +326,7 @@ typedef struct {
   uv_async_cb async_cb;                                                       \
   void* queue[2];                                                             \
   int pending;                                                                \
+  int busy;                                                                   \
```

Comment:
Adding a new field here breaks the ABI, at least on 32-bit platforms :/

I guess we could try to re-use one of the `reserved` slots from `UV_HANDLE_FIELDS` for this? I’m not sure what the intention there is in terms of future extensibility, but I think it should be okay to add another member to the union, e.g. a struct that has both `fd` and `busy`?

Reply:
My production environment is ARMv7l 32bit platform. I'm a bit confused
about 'breaks the ABI', can you explain more, thanks.

Reply:
Anything that changes `sizeof(uv_async_t)` is off the table for the v1.x branch because that means upgrading the system's libuv.so would break existing programs. They would need to be recompiled.

Reply:
Thanks, I understand now.
To not break the ABI, can I use `fd` as `busy` in union according to @addaleax 's suggestion.
I found libuv only use this `fd`  in windows platform.

> Adding a new field here breaks the ABI, at least on 32-bit platforms :/
> 
> I guess we could try to re-use one of the `reserved` slots from `UV_HANDLE_FIELDS` for this? I’m not sure what the intention there is in terms of future extensibility, but I think it should be okay to add another member to the union, e.g. a struct that has both `fd` and `busy`?

Reply:
I update the PR to use handle->u.fd as busy flag.